main/tests/server/src/com/google/refine/operations/recon/ExtendDataOperationTests.java

```diff
@@ -249,7 +249,7 @@ public void testFetchStrings() throws Exception {
       		  "{"
   			+ "\"rows\": {"
   			+ "    \"Q794\": {\"P297\": [{\"str\": \"IR\"}]},"
-  			+ "    \"Q863\": {\"P297\": [{\"str\": \"TJ\"}]},"
+  			+ "    \"Q863\": {\"P297\": []},"
```

Comment:
This test was changed to make sure the data extension operation does not create non-null cells with null values.